build.sh

```diff
@@ -494,12 +495,20 @@ if (( ${NUMARGS} == 0 )) || hasArg go; then
 fi
 
 # Build the cuvs Java bindings
+SKIP_JAVA_TESTS=""
+for arg in "$@"
+do
+  if [ "$arg" == "--skip-java-tests" ]; then
+    SKIP_JAVA_TESTS="--skip-java-tests"
+  fi
+done
+
 if (( ${NUMARGS} == 0 )) || hasArg java; then
     if ! hasArg libcuvs; then
         echo "Please add 'libcuvs' to this script's arguments (ex. './build.sh libcuvs java') if libcuvs libraries are not already built"
     fi
     cd ${REPODIR}/java
-    ./build.sh
+    ./build.sh $SKIP_JAVA_TESTS
```

Comment:
```suggestion
    SKIP_JAVA_TESTS="${SKIP_JAVA_TESTS}" ./build.sh
```

As currently written, this is not going to work. `java/build.sh` on this branch is using `$SKIP_JAVA_TESTS` and not looking for any arguments passed positionally.

Reply:
Yh Good eyesight! I found it was removed by last update, and it will be added back.

Reply:
Done